orienteer-bpm/src/main/java/org/orienteer/bpm/Initializer.java

```diff
@@ -18,6 +18,7 @@ public void init(Application application) {
 	@Override
 	public void destroy(Application application) {
 		OrienteerWebApplication app = (OrienteerWebApplication)application;
+		app.unregisterModule(BPMModule.class);
```

Comment:
Question: is it required to unregister module here? Something will go wrong if it's not called?